client/components/domains/register-domain-step/index.jsx

```diff
@@ -406,6 +406,7 @@ class RegisterDomainStep extends React.Component {
 							describedBy={ 'step-header' }
 							dir="ltr"
 							initialValue={ this.state.lastQuery }
+							inputLabel={ 'What would you like your domain name to be?' }
```

Comment:
This is missing a translate function. 
```suggestion
							inputLabel={ this.props.translate( 'What would you like your domain name to be?' ) }
```